src/Mailgun/Api/MailingList.php

```diff
@@ -0,0 +1,268 @@
+<?php
+
+/*
+ * Copyright (C) 2013 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Api;
+
+use Mailgun\Assert;
+use Mailgun\Model\MailingList\PagesResponse;
+use Mailgun\Model\MailingList\CreateResponse;
+use Mailgun\Model\MailingList\ShowResponse;
+use Mailgun\Model\MailingList\UpdateResponse;
+use Mailgun\Model\MailingList\DeleteResponse;
+use Mailgun\Model\MailingList\MembersResponse;
+use Mailgun\Model\MailingList\CreateMemberResponse;
+use Mailgun\Model\MailingList\ShowMemberResponse;
+use Mailgun\Model\MailingList\UpdateMemberResponse;
+use Mailgun\Model\MailingList\DeleteMemberResponse;
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * @author Michael Münch <helmchen@sounds-like.me>
+ */
+class MailingList extends HttpApi
+{
+    /**
+     * Returns a paginated list of mailing lists on the domain.
+     *
+     * @param int $limit Maximum number of records to return (optional: 100 by default)
+     *
+     * @return PagesResponse|array|ResponseInterface
+     *
+     * @throws \Exception
+     */
+    public function pages($limit = 100)
+    {
+        Assert::integer($limit);
+
+        $params = compact('limit');
```

Comment:
Although it might work, I would say to create the `$params` array as an array, like we have on other API files, so be consistent with the code. Everybody know what is `array()` or `[]` (ideally `[]` due it's the one set by StyleCI) but not everybody might know what is `compact`.

Reply:
I believe that this was added to re-use variable names, like `$access_level` (comments below) however PHP variable names should be written as `camelCase`